org.lflang/src/org/lflang/generator/ts/TSReactionGenerator.kt

```diff
@@ -449,6 +500,7 @@ class TSReactionGenerator(
         // If the app is federated, only generate
         // reactions that are contained by that federate
         val generatedReactions: List<Reaction>
+        val generatedMutations: List<Reaction>
```

Comment:
Why is the variable called `generatedMutations`? I think just `mutations` is more accurate.

Reply:
I changed the variable name! https://github.com/lf-lang/lingua-franca/commit/b349e6010a667132c40f78a71128e268fecd00dc